packages/core/src/components/Dialog/__stories__/Dialog.stories.js

```diff
@@ -19,14 +20,30 @@ const metaSettings = createStoryMetaSettingsDecorator({
   actionPropsArray: [] // List the component's actions here
 });
 
+const showHideArgTypes = {
+  options: Object.values(HideShowEvent),
+  control: {
+    type: "multi-select"
+  },
+  table: {
+    type: {
+      summary: Object.values(HideShowEvent).join(" | ")
+    }
+  }
+};
```

Comment:
definitely should be (in a future enhancement, maybe create a task?) a common util, or as part of the `createMeta`

Reply:
I'll create a task for it in the backlog